src/webgpu/api/validation/validation_test.ts

```diff
@@ -324,9 +324,9 @@ export class ValidationTest extends GPUTest {
       occlusionQuerySet?: GPUQuerySet;
     } = {}
   ): CommandBufferMaker<T> {
-    const fullAttachmentInfo = {
+    const fullAttachmentInfo: GPURenderBundleEncoderDescriptor = {
       // Defaults if not overridden:
-      colorFormats: ['rgba8unorm'],
+      colorFormats: ['rgba8unorm'] as GPUTextureFormat[],
```

Comment:
Oops, this breakage is because the type changed from `Iterable<GPUTextureFormat>` to `Array<GPUTextureFormat>`. That should be fixed in the types.

Reply:
Oops! Sorry! https://github.com/gpuweb/types/pull/79

Reply:
No worries!